src/styles/isomer-template/components/_cards.scss

```diff
@@ -0,0 +1,98 @@
+@use "sass:map";
```

Comment:
casual reminder to duplicate over to template as the styles need to be replicated

Reply:
skimming, didn't have issues based off what i saw. will leave design to review

Reply:
Yup, design reviewed the template earlier.